Library/Formula/rabbitmq-c.rb

```diff
@@ -31,10 +29,10 @@ def install
     args << "-DBUILD_API_DOCS=OFF"
 
     args << if build.with? "tools"
-      "-DBUILD_TOOLS=ON"
-    else
-      "-DBUILD_TOOLS=OFF"
-    end
+              "-DBUILD_TOOLS=ON"
```

Comment:
Indentation has slipped way off here. Suspect you've indented with two tabs instead of two spaces.